1_preprocessing_data/PyBaSiC_Pipelines/Illumination_Correction.py

```diff
@@ -29,49 +28,61 @@
 # In[2]:
 
 
-def load_pybasic_data(channel_path: pathlib.Path, channel: str, file_extension=".tif"):
+def load_pybasic_data(channels_path: pathlib.Path, channel: str, file_extension=".tif", verbosity: str = True):
     """
     Load all images from a specified directory in preparation for pybasic illum correction
 
     Parameters:
-    channel_path (pathlib.Path): The directory where the channel folders are stored
+    channels_path (pathlib.Path): path to directory where the all images for each channel are stored
     channel (str): The name of the channel (currently one of ["GFP", "RFP", "DAPI"])
     file_extension (str): The filename extension of the types of files to search for (default: ".tif")
-    
+    verbosity (str): Prints out information regarding the function running through the images (default: "True")
+
     Returns:
         channel_images: list of ndarrays of the loaded in images
         image_files: list of strings of the paths to each image
     """
-    # Creates a full pathlib.Path using the specified path to directory with channel folders and one of the channels that you designate
-    full_channel_path = pathlib.Path(channel_path, channel)
-
-    # For loop that iterates through the channel directory, coverts image paths to strings, finds strings with designated file extension (in this case the images are
-    # of type .tif), strips the file extension from path, sorts the images to be in alphabetically order, and then loads in the data using the PyBaSiC function
-    image_files = []
-    for image_file in full_channel_path.iterdir():
-        image_file = str(image_file)
-        if image_file.endswith(file_extension):
-            image_files.append(image_file.strip(file_extension))
+    image_files = [] # List that holds all of the paths to the images in the directory for a specific channel
+    image_names = [] # List that holds the names (str) for the images that is used for saving 
+    images = [] # List of numpy arrays for the images that are read in
+
+    # This for loop will run through the specified directory, find images for a specific channel (channel name is in the file name metadata), save paths to a list, and then save the names 
+    # to a list after stripping the file_extension
+    for image_path in channels_path.iterdir():
+        image_path = str(image_path)
+        if channel in image_path:
+            image_files.append(image_path)
+            if image_path.endswith(file_extension):
+                image_names.append(image_path.strip(file_extension))
    
+    # Sorts the file paths and names into alphabetical order (puts well C at the start)
     image_files.sort()
-    channel_images = pybasic.tools.load_data(full_channel_path, file_extension, verbosity = True)
+    image_names.sort()
```

Comment:
I think this is duplicated by mistake, right?

Reply:
Talked about this, all good!